schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT NOT NULL,
```

Comment:
- [ ] [OPTIONAL] You could optionally make the ID as auto-incremented and the primary key.

Reply:
Hi, @Nemwel-Boniface Thanks for the detailed review, I already have the queries in my queries.sql file I am not sure what you meant by I am supposed to copy my queries again. Can you please explain further, Thank you

Reply:
Hello @niicode I did not quite get you here. But I can see that you included the query results screenshots in your pull request description which are perfectly fine.